topics/single-cell/tutorials/bulk-deconvolution-evaluate/tutorial.md

```diff
@@ -0,0 +1,585 @@
+>    - {% icon param-file %} *"Input Dataset"*: `RData ESet Object` (output of **Construct Expression Set Object** {% icon tool %})
+>    - *"Size of output colection"*: `20`
+>
+> 2. **Rename** {% icon galaxy-pencil %} output `ESet Object`
+>
+{: .hands_on}
+
+
+# Create pseudo-bulk and actual cell proportions
+
+We are now going to run our first workflow! This workflow will extract a subsample from the data containing 200 random cells. The workflow will then perform two things with this subsample:
+
+1. Count the cell types and proportions of the data in order to be used as reference later against the predicted proportion values
+2. Remove the cell types and convert the single-cell data into pseudo-bulk data to be later inputted into the deconvolution tools.
+
+The above will be done twice to emulate multiple "subjects". Since the deconvolution tools will be expecting the bulk-RNA data to comprise of at least 2 subjects (each with their own bulk data). For this tutorial our subjects will simply be called **A** and **B**. However, in the real world these subjects could be different patients, tissue samples, diseased/healthy, etc. 
+
+> <comment-title>Different Results</comment-title>
+> Note that since we are selecting 20 samples, each containing 200 randomly selected cells. The plots and results presented in this tutorial will differ from your own. There will be some similarities such as certain cells being in higher proportion to others but the exact values with differ!
+{: .comment}
+
+**Remember** since we have a collection of 20 inputs, the output of this workflow will be a collection of 20 elements, each corresponding to the input elements. Each output will have its own random selection of 200 cells.
+
+> <comment-title>Inputting multiple datasets</comment-title>
+>
+> In order to upload the input collections into the workflow, you first need to set the input type to **Multiple datasets** in the input file selection.
+> 
+> ![Multiple Datasets](../../images/bulk-deconvolution-evaluate/batch-mode.png "Multiple Datasets button in Galaxy")
+{: .comment}
+
+> <hands-on-title>Run pseudo-bulk and actual proportions workflow</hands-on-title>
+>
+> 1. **Import the workflow** into Galaxy
+>    - Copy the URL (e.g. via right-click) of [this workflow](https://usegalaxy.eu/u/hexhowells/w/deconv-eval-stage-1) or download it to your computer.
+>    - Import the workflow into Galaxy
+>
+>    {% snippet faqs/galaxy/workflows_run_trs.md path="topics/transcriptomics/tutorials/rna-seq-reads-to-counts/workflows/qc_report.ga" title="QC Report" %}
+>
+> 2. Run **Workflow pseudobulk and actual proportions** {% icon workflow %} using the following parameters:
+>    - {% icon param-collection %} *"Metadata"*: `Metadata`
+>    - {% icon param-collection %} *"Expression Data"*: `Expression Data`
+>
+>    {% snippet faqs/galaxy/workflows_run.md %}
+> 3. Add a tag labelled `#A` to the first "Actual cell proportions" and "Pseudobulk" collections

```

Comment:
I am not completely sure, but I feel like the term "actual cell proportions" might be a little misleading. The cell proportions, as indicated by proportional representation in the single-cell data, are often different from the true in vivo cell type proportions due to systematic drop out biases during data collection. This might be worth mentioning, or maybe a different term which doesn't use "actual" could be substituted.